lib/Loader.php

```diff
@@ -427,7 +427,18 @@ public function get_twig() {
 				return null;
 			}));
 		}
-		$twig->addExtension($this->_get_cache_extension());
+		/**
+		 * Filters the cache extension activation
+		 *
+		 * Allows users to disable the cache extension and use their own
+		 *
+		 * @since 2.0.0
+		 * @param bool $enable_cache_extension
+		 */
+		$enable_cache_extension = apply_filters('timber/enable_cache_extension', true);
```

Comment:
Because other hooks that are related to the cache all use the `timber/cache/…` namespace, I propose to use it here as well.

```suggestion
		$enable_cache_extension = apply_filters('timber/cache/enable_extension', true);
```